src/annotations/api/index.ts

```diff
@@ -0,0 +1,27 @@
+// Types
+import {PostAnnotationResponse, PostAnnotationPayload} from 'src/types'
+
+// Libraries
+import axios, {AxiosResponse} from 'axios'
+
+// Constants
+import {API_BASE_PATH} from 'src/shared/constants'
+
+// URL
+const url = `${API_BASE_PATH}api/v2private/annotations`
+
+export const writeAnnotation = async (
+  annotations: PostAnnotationPayload[]
+): Promise<AxiosResponse<PostAnnotationResponse[]>> => {
+  try {
+    const res = await axios.post(url, annotations)
+
+    if (res.status !== 200) {
+      throw new Error(res.data?.message)
+    }
+
+    return res.data
+  } catch (err) {
+    throw new Error(err?.message)
+  }
```

Comment:
✂️ 

this is the same as not having a catch block because nothing is being done with the error except re-throwing. i liked, it's actually worse than not having a catch block because this strips the error object of context and information, and it also creates a temporary variable that it copies from the caught error.

since this method throws an error when the status is less than 300, any code using this method will have to handle an error. since it'll have to handle an error, it can handle this error that you're catching and re-throwing, and probably know what to do with the error better than this code does. this is api code, it doesn't know what it **means** to have an error, just that something didn't work.

if you're not going to do anything with an error, don't catch it.